Modules/ModParser.lua

```diff
@@ -258,6 +258,7 @@ local modNameList = {
 	["recovery rate of life and energy shield"] = { "LifeRecoveryRate", "EnergyShieldRecoveryRate" },
 	["maximum life, mana and global energy shield"] = { "Life", "Mana", "EnergyShield", tag = { type = "Global" } },
 	["non-chaos damage taken bypasses energy shield"] = {"PhysicalEnergyShieldBypass", "LightningEnergyShieldBypass", "ColdEnergyShieldBypass", "FireEnergyShieldBypass" },
+	["damage from hits is taken from your spectres' life before you"] = "DamageFromHitsTakenFromSpectres",
```

Comment:
I haven't tested this, but see if this works instead

```suggestion
	["damage from hits is taken from your spectres' life before you"] = { "DamageTakenWhenHit", tag = { type = "SkillName", skillName = "Raise Spectre" } },
```